github/store/db.go

```diff
@@ -202,6 +202,8 @@ func (s *DB) SaveUser(user *graphql.UserExtended) error {
 	hash := sha256.Sum256([]byte(st))
 	hashString := fmt.Sprintf("%x", hash)
 
+	fmt.Printf("%+v\n", user)
```

Comment:
let's replace it to `log.Infof` or `log.Debugf`

Reply:
I actually erased it.